apps/src/applab/applab.js

```diff
@@ -1491,6 +1513,7 @@ Applab.execute = function() {
       'if (obj) { var ret = obj.fn.apply(null, obj.arguments ? obj.arguments : null);' +
                  'setCallbackRetVal(ret); }}';
     var session = studioApp.editor.aceEditor.getSession();
+    annotationList.attachToSession(session);
```

Comment:
Yeah, they should be. I guess I could do a strict equality check and throw if we set it to something different.